src/components/Modals/Send/utils.ts

```diff
@@ -116,13 +116,13 @@ export const handleSend = async ({
     const acccountMetadataFilter = { accountId: sendInput.accountId }
     const accountMetadata = selectPortfolioAccountMetadataByAccountId(state, acccountMetadataFilter)
     const snapId = getConfig().REACT_APP_SNAP_ID
-    const isSnapsInstalled = await shapeShiftSnapInstalled(snapId)
+    const isSnapInstalled = await shapeShiftSnapInstalled(snapId)
```

Comment:
OCD rename - duplicated logic bit in `useStakingAction`.
Frontrunning the review comments here, we should *not* run this (both here  and in `useStakingAction`) in case the HDWallet isn't the MM multichain one / MM isn't *actual* MM, but these are improved in https://github.com/shapeshift/web/pull/5287, not the scope of this PR

Reply:
I should have gotten MEV protection.